### ISSUE TITLE
Rosella run questions

### ISSUE THREAD

**Issue by Valentin-Bio-zz** [rhysnewell/rosella#28]
Hello I'm trying to run rosella on a HPC server, I installed the program via conda 

`conda install -c bioconda rosella `

once installed I ran `rosella bin` as mentioned on documentation but the program is not accepting the 'bin' command:

`rosella bin -r final.contigs.fa --coverage-values depth.txt -o rosella_bins -t 32`

I got this error message: 

> error: Found argument 'bin' which wasn't expected, or isn't valid in this context

I'm using the depth.txt file built from metabat2 `jgi_summarize_bam_contig_depths` program as the coverage file.

**Comment by rhysnewell** [comment]
Ah yes sorry, my bad. The bin command got changed to recover. So just take your current command and replace bin with recover

**Comment by Valentin-Bio** [comment]
Hello Rhys, thanks for the clarification!. 

Now I run Rosella and got this:

```
[2022-11-12T20:19:22Z INFO  rosella] rosella version 0.4.2
[2022-11-12T20:19:22Z INFO  rosella] Using min-covered-fraction 0%
[2022-11-12T20:19:22Z INFO  rosella] Using min-read-aligned-percent 0%
[2022-11-12T20:19:26Z ERROR rosella::external_command_checker] Could not find an available flight executable.
[2022-11-12T20:19:26Z ERROR rosella::external_command_checker] The STDERR was: "bash: flight: command not found\n"
thread 'main' panicked at 'Cannot continue without flight. Testing for presence with `flight bin -h` failed', src/external_command_checker.rs:103:9
```

**Comment by rhysnewell** [comment]
Did you run this line from the install instructions: `pip install git+https://github.com/rhysnewell/flight.git#egg=flight-genome`

**Comment by Valentin-Bio** [comment]
Oh i see, I have problems installing flight. I have pip 8.1.2 and cannot upgrade it. is there a conda way to install flight? or a way in which I can avoid pip? 

thanks for your fast answer

**Comment by rhysnewell** [comment]
If you installed `rosella` via conda shouldn't you be able to install a new version of `pip` via conda as well, right? If not, you could also try installing flight from source: https://github.com/rhysnewell/flight 
Doing so would also require using `pip` but you might be able to get away with using an older version

**Comment by thulasis** [comment]
Same error as reported earlier and unable to install flight as well/

**Comment by rhysnewell** [comment]
Hey there,

So what error do you get when you try to install flight? What version of python are you running, and are you in installing into a conda or virtual environment?

Cheers,
Rhys

**Comment by aljazdzy** [comment]
I'm also interested in if this was resolved as I have been having similar issues.  I'll be back if I figure it out.

**Comment by TJrogers86** [comment]
> recover

Could you update this in the READ ME file? I had the same error and had to hunt for the solution. It might save you some hassle if you start getting this question over and over lol

**Comment by rhysnewell** [comment]
Should be fixed `v0.5.0`
